build.gradle.kts

```diff
@@ -222,74 +192,33 @@ if (project.hasProperty("enableCodeStyle")) {
 }
 
 println("Using Kotlin compiler version: ${org.jetbrains.kotlin.config.KotlinCompilerVersion.VERSION}")
-if (build_snapshot_train) {
-    println("Hacking test tasks, removing stress and flaky tests")
-    allprojects {
-        tasks.withType<Test>().all {
-            exclude("**/*ServerSocketTest*")
-            exclude("**/*NettyStressTest*")
-            exclude("**/*CIOMultithreadedTest*")
-            exclude("**/*testBlockingConcurrency*")
-            exclude("**/*testBigFile*")
-            exclude("**/*numberTest*")
-            exclude("**/*testWithPause*")
-            exclude("**/*WebSocketTest*")
-            exclude("**/*PostTest*")
-            exclude("**/*testCustomUrls*")
-            exclude("**/*testStaticServeFromDir*")
-            exclude("**/*testRedirect*")
-            exclude("**/*CIOHttpsTest*")
-        }
-    }
-
-    println("Manifest of kotlin-compiler-embeddable.jar")
+filterSnapshotTests()
 
-    subprojects.filter { it.name == "ktor-client" }.forEach {
-        configurations.matching { it.name == "kotlinCompilerClasspath" }.all {
-            resolvedConfiguration.files.filter { it.name.contains("kotlin-compiler-embeddable") }.forEach {
-                val manifest = zipTree(it).matching {
-                    include("META-INF/MANIFEST.MF")
-                }.files.first()
-
-                manifest.readLines().forEach {
-                    println(it)
-                }
-            }
-        }
-    }
+if (project.hasProperty("enable-coverage")) {
+    apply(from = "gradle/jacoco.gradle")
 }
 
-afterEvaluate {
-    val allCompileKotlinTasks = subprojects.mapNotNull {
-        if (it.hasProperty("compileKotlinJvm")) {
-            it.property("compileKotlinJvm")
-        } else null
-    }
-
-    tasks.create<org.jetbrains.dokka.gradle.DokkaTask>("dokkaWebsite") {
-        outputFormat = "kotlin-website"
-        outputDirectory = "${rootProject.projectDir}/apidoc"
+plugins {
+    id("org.jetbrains.dokka") version "1.4.32"
+}
 
-        kotlinTasks(closureOf<Any?> { allCompileKotlinTasks })
+subprojects {
+    plugins.apply("org.jetbrains.dokka")
 
-        reportUndocumented = false
+    tasks.withType<DokkaTaskPartial> {
+        dokkaSourceSets.configureEach {
+            if (platform.get().name == "js") {
+                suppress.set(true)
+            }
+        }
     }
 }
 
-if (project.hasProperty("enable-coverage")) {
-    apply(from = "gradle/jacoco.gradle")
-}
-
-// Disable binary compatibility check for JVM IR compiler output by default
-val jvm_ir_enabled: Boolean by extra
-val jvm_ir_api_check_enabled: Boolean by extra
+println("${rootDir.absolutePath}/docs")
```

Comment:
Please remove this `println` or add more information to the log string

Reply:
Thanks, will remove